workloads/etcd-perf/run_etcd_tests_fromgit.sh

```diff
@@ -42,35 +43,44 @@ cat << EOF | oc create -n my-ripsaw -f -
 apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
 kind: Benchmark
 metadata:
-  name: fio-benchmark
+  name: etcd-fio
   namespace: my-ripsaw
 spec:
   elasticsearch:
-    server: $_es
-    port: $_es_port
-  clustername: $cloud_name
+    server: ${_es}
+    port: ${_es_port}
+  clustername: ${cloud_name}
   test_user: ${cloud_name}-ci
   metadata_collection: true
   metadata_sa: backpack-view
   metadata_privileged: true
+  hostpath: /var/lib/fio-etcd
   workload:
-    name: byowl
+    name: fio_distributed
     args:
-      image: "quay.io/cloud-bulldozer/fio"
-      clients: 1
-      commands: |
-        for i in $(seq ${samples} | xargs); do
-          mkdir -p /tmp/perf;
-          fio --rw=write --ioengine=sync --fdatasync=1 --directory=/tmp/perf --size=22m --bs=2300 --name=test;
-          sleep 5;
-        done
+      iodepth: 1
+      log_sample_rate: 1000
+      samples: 5
+      servers: 1
+      jobs:
+        - write
+      bs:
+        - 2300
+      numjobs:
+        - 1
+      filesize: 22Mib
+  global_overrides:
```

Comment:
I assume this is supposed to be aligned under workload?

Reply:
fixed!